src/utils/daos_hdlr.c

```diff
@@ -2649,15 +2653,15 @@ dm_connect(struct cmd_args_s *ap,
 					    dst_cont_info, NULL);
 		else
 			rc = -DER_NONEXIST;
+
 		if (rc == -DER_NONEXIST) {
 			uuid_t cuuid;
 
 			if (ca->cont_layout == DAOS_PROP_CO_LAYOUT_POSIX) {
+				/** XXX: should copy dfs_attr & properties from source container */
```

Comment:
@johannlombardi this functionally is being added in PR #6617

Reply:
Sounds good.